scg-system/src/main/java/io/telicent/core/SmartCacheGraph.java

```diff
@@ -119,4 +132,51 @@ private static boolean isAuthEnabled() {
     private static boolean isInitialCompactionEnabled() {
         return !Configurator.get(FMod_InitialCompaction.DISABLE_INITIAL_COMPACTION, Boolean::parseBoolean, false);
     }
+
+    private static String convertToAbsolutePathURI(String fileUri) {
+        // Remove the "file:" prefix to get the relative path
+        String relativePath = fileUri.substring(5);
+
+        // Convert the relative path to an absolute path
+        File file = new File(relativePath);
+        String absolutePath = file.getAbsolutePath();
+
+        // Return the absolute path as a file URI
+        return "file:/" + absolutePath;
+    }
+
+    private static void convertYamlConfigToRDF(String... args) {
+        YAMLConfigParser ycp = new YAMLConfigParser();
+        RDFConfigGenerator rcg = new RDFConfigGenerator();
+        String configPath;
+        String pattern = ".*\\.(yaml|yml)$";
+        Pattern regex = Pattern.compile(pattern);
+        for (int i = 0; i < args.length; i++) {
+            if (args[i].equalsIgnoreCase("--conf") && i + 1 < args.length ) {
+                configPath = args[i + 1];
+                Matcher matcher = regex.matcher(configPath);
+                if (matcher.matches()) {
+                    try {
+                        ConfigStruct configStruct = ycp.runYAMLParser(configPath);
+                        Model configModel = rcg.createRDFModel(configStruct);
+
+                        File rdfConfigPath = File.createTempFile("RDF.state", ".ttl");
```

Comment:
Can we give this a more informative prefix e.g. `generated-config-` as the current one doesn't seem to match the purpose